### PR TITLE
Set flytekit<2.0 in plugins

### PR DESCRIPTION
**Pull request by eapolinario**
Signed-off-by: Eduardo Apolinario <eapolinario@users.noreply.github.com>

# TL;DR
Change the upper bound flytekit version in plugins to '<2.0.0' 

## Type
 - [x] Bug Fix
 - [ ] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
The official flytekit docker images failed to build when we released the beta version flytekit 1.2.0b0 [last week](https://github.com/flyteorg/flytekit/runs/7887779364?check_suite_focus=true) because of this error:
```
ERROR: Cannot install flytekit==1.2.0b0 and flytekitplugins-pod==1.2.0b0 because these package versions have conflicting dependencies.

The conflict is caused by:
    The user requested flytekit==1.2.0b0
    flytekitplugins-pod 1.2.0b0 depends on flytekit<1.2.0 and >=1.1.0b0
```

## Tracking Issue
https://github.com/flyteorg/flyte/issues/2855

## Follow-up issue
_NA_

### CODE REVIEW

**Review by wild-endeavor — APPROVED** [review]
🤞
